WordPress/Classes/Models/Post.m

```diff
@@ -196,18 +196,19 @@ - (NSString *)contentPreviewForDisplay
 
 - (NSString *)statusForDisplay
 {
-    NSString *statusString = [self statusTitle];
-    if ([self.status isEqualToString:PostStatusPublish] || [self.status isEqualToString:PostStatusDraft]) {
-        statusString = nil;
-    }
-    if (![self hasRemote]) {
+    NSString *statusString = nil;
+    
+    if (self.isRevision) {
         NSString *localOnly = NSLocalizedString(@"Local", @"A status label for a post that only exists on the user's iOS device, and has not yet been published to their blog.");
         if (statusString) {
             statusString = [NSString stringWithFormat:@"%@, %@", statusString, localOnly];
         } else {
             statusString = localOnly;
```

Comment:
Should the status we show make a distinction between posts that are "local only" and posts that have local changes? Just a thought.